app/models/event.rb

```diff
@@ -62,4 +71,9 @@ def get_remote_participation
   def get_total_participation
     invites.where(status: "Accepted").count + invites.where(status:"Accepted[remote]").count
   end
+
+  def future_date?
+    errors.add(:start_time, "can't be in the past") unless
+      start_time.future?
+  end
```

Comment:
Ah, we're not using the `future_date?` method anymore now that we have the `FutureTime` validator, can you remove this method and repush?